tests/func/experiments/test_show.py

```diff
@@ -582,3 +582,74 @@ def test_show_only_changed(tmp_dir, dvc, scm, capsys):
     cap = capsys.readouterr()
 
     assert "bar" not in cap.out
+
+
+def test_show_parallel_coordinates(tmp_dir, dvc, scm, mocker):
+    from dvc.command import experiments
+
+    webbroser_open = mocker.patch("webbrowser.open")
+    show_experiments = mocker.spy(experiments, "show_experiments")
+
+    tmp_dir.gen("copy.py", COPY_SCRIPT)
+    params_file = tmp_dir / "params.yaml"
+    params_data = {
+        "foo": 1,
+        "bar": 1,
+    }
+    (tmp_dir / params_file).dump(params_data)
+
+    dvc.run(
+        cmd="python copy.py params.yaml metrics.yaml",
+        metrics_no_cache=["metrics.yaml"],
+        params=["foo", "bar"],
+        name="copy-file",
+        deps=["copy.py"],
+    )
+    scm.add(
+        [
+            "dvc.yaml",
+            "dvc.lock",
+            "copy.py",
+            "params.yaml",
+            "metrics.yaml",
+            ".gitignore",
+        ]
+    )
+    scm.commit("init")
```

Comment:
How much do we need `bar`? Seems like besides that this is identical with `exp_stage` fixture

Reply:
I think I was going to use it for checking that `--only-changed` is implicitly called when `--html` is passed so `bar` should not be present in the table

Reply:
I need to revise the tests as I don't see that check 😅